orchagent/orchdaemon.cpp

```diff
@@ -958,6 +963,20 @@ void OrchDaemon::addOrchList(Orch *o)
     m_orchList.push_back(o);
 }
 
+void OrchDaemon::heartBeat(std::chrono::time_point<std::chrono::high_resolution_clock> tcurrent)
+{
+    static auto tlast = std::chrono::high_resolution_clock::now();
```

Comment:
>static [](http://example.com/codeflow?start=4&length=6)

You are assuming `OrchDaemon` has only single instance in the process. To be super safe, you can use a static member variable instead of a static  function variable.

Reply:
Fixed, change to a member variable.

Reply:
Fixed, change to none static member.